changelog.md

```diff
@@ -1,4 +1,8 @@
 
+1.1.4.0
+
+* Add --pattern and --ipattern selectors.
```

Comment:
Shouldn't this be just `pattern` and `ipattern`? These aren't full options after all.